src/layouts/default.hbs

```diff
@@ -115,10 +114,10 @@
 						<div class="site-footer-unit">
 							<div class="site-footer-unit-inner">
 								<div class="footer-top-links">
-									<div class="footer-title mb-16">Enterprise</div>
+									<div class="footer-title mb-16">Services</div>
```

Comment:
Not a fan of calling support a "service" - at this time it's free!

Reply:
How about we replace the `Contact Us` email link with the link to the new support page, but keep the same name. And then enterprise can stay it's own section.

Reply:
Actually wait, how about this:

ABOUT
- Blog
- Our Team
- Donate

POLICIES
- Terms of Service
- Privacy
- Security

GET IN TOUCH
- Enterprise Partnerships
- Support

Reply:
I like it.

`Support` -> `Contact`?

Also, if you can think of any 1-word titles I think that would flow well.. Thoughts?

Reply:
Alright, we're good for final review.